cdap-ui/app/cdap/components/SplashScreen/index.js

```diff
@@ -50,16 +51,18 @@ import T from 'i18n-react';
   componentWillMount() {
     MyUserStoreApi.get().subscribe((res) => {
       this.setState({
-        showSplashScreen : !res.property["standalone-welcome-message"]
+        showSplashScreen : window.CDAP_CONFIG.cdap.standaloneWebsiteSDKDownload && !window.CDAP_CONFIG.isEnterprise && res.property["standalone-welcome-message"],
```

Comment:
We could check for the condition and its `true` then we can call the setState. This could avoid re-rendering the component even though its value is the same. (Not sure if react already does this but still could avoid a function call and check).